esphome/components/daly_bms/daly_bms.h

```diff
@@ -110,6 +111,13 @@ class DalyBmsComponent : public PollingComponent, public uart::UARTDevice {
 
   binary_sensor::BinarySensor *charging_mos_enabled_{nullptr};
   binary_sensor::BinarySensor *discharging_mos_enabled_{nullptr};
+
+  std::vector<uint8_t> data_;
+  bool receiving_{false};
+  uint8_t data_count_;
+  uint32_t last_transmission_{0};
+  bool trigger_next;
+  uint8_t next_request;
```

Comment:
trailing underscore for protected members
```suggestion
  bool trigger_next_;
  uint8_t next_request_;
```